docs/how-to-guides/upgrade-bootstrap-kubelet.md

```diff
@@ -0,0 +1,44 @@
+# How to upgrade bootstrap kubelet
+
+## Introduction
+
+Bootstrap kubelet is the kubelet running on the node as a systemd service. Unlike most components on the cluster, lokoctl cannot update it. So there are some manual steps involved in updating the bootstrap kubelet.
+
+## Steps
+
+Apply the following steps to all the nodes one at a time.
+
+### Drain the node
+
+```
+kubectl drain <node name>
+```
+
+### Find out the IP
```

Comment:
This section says find the IP address, but then tells to SSH into the node. We should be more explicit.